app/controllers/schools/add_participants_controller.rb

```diff
@@ -11,8 +11,7 @@ class AddParticipantsController < ::Schools::BaseController
     result as: :participant_profile
 
     abandon_journey_path do
-      participants = User.is_participant.in_school(@school.id)
-      participants.any? ? schools_participants_path : schools_cohort_path
+      @school_cohort.active_participants.any? ? schools_participants_path : schools_cohort_path
```

Comment:
This will break if they have an NPQ ppt

Reply:
ehh, I really should call the scopes `ecf_participants` and `active_ecf_participants`. I hate NPQ profiles being participants.